src/middlewared/middlewared/plugins/pool.py

```diff
@@ -2125,7 +2122,6 @@ async def lock(self, id, options):
         Dict(
             'unlock_options',
             Bool('key_file', default=False),
-            Bool('toggle_attachments', default=True),
```

Comment:
You removed this but I see a restart_attachmantes was "replaced" with? Where is it now?